### PR TITLE
refactor: move event and person query to /posthog

### PR DESCRIPTION
**Pull request by EDsCODE**
## Problem

- continue moving code to proper places
- move person_query, event_query, and various utils to /posthog
- need to merge this first https://github.com/PostHog/posthog-cloud/pull/177

## Changes

👉 *Stay up-to-date with our [coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: ee/clickhouse/queries/breakdown_props.py [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
 from ee.clickhouse.models.cohort import format_filter_query
 from ee.clickhouse.models.entity import get_entity_filtering_params
 from ee.clickhouse.models.property import (
     get_property_string_expr,
     get_single_or_multi_property_string_expr,
     parse_prop_grouped_clauses,
 )
-from ee.clickhouse.models.util import PersonPropertiesMode
-from ee.clickhouse.queries.column_optimizer import ColumnOptimizer
+from ee.clickhouse.queries.column_optimizer import EE_ColumnOptimizer
```
Comment (neilkakkar) [inline ee/clickhouse/queries/breakdown_props.py:10]: I assume EE_ColumnOptimizer will be gone at the end of all these PRs?

File: ee/clickhouse/queries/event_query.py [hunk 2]
```diff
@@ -1,39 +1,21 @@
...
+from posthog.models.property import PropertyName
 from posthog.models.team import Team
+from posthog.queries.event_query import EventQuery
 
 
-class ClickhouseEventQuery(metaclass=ABCMeta):
-    DISTINCT_ID_TABLE_ALIAS = "pdi"
-    PERSON_TABLE_ALIAS = "person"
-    EVENT_TABLE_ALIAS = "e"
+class EE_EventQuery(EventQuery):
```
Comment (neilkakkar) [inline ee/clickhouse/queries/event_query.py:16]: .. and same for everything following this pattern?

... or is this more like a new pattern, where EE related code for that specific class is named like so?

File: ee/clickhouse/queries/breakdown_props.py (hunk 1 above)
Comment (EDsCODE) [reply ee/clickhouse/queries/breakdown_props.py:10]: Nope, to adhere to the principle of FOSS, keeping as much of the group logic in the /ee directory as possible

Comment (neilkakkar) [reply ee/clickhouse/queries/breakdown_props.py:10]: Ah, I see, right, makes sense!

Can I propose renaming all of these to:

`EnterpriseColumnOptimizer`, `EnterpriseEventQuery` , etc. etc.

File: ee/clickhouse/queries/event_query.py (hunk 2 above)
Comment (EDsCODE) [reply ee/clickhouse/queries/event_query.py:16]: haha yeah let's do that

**Review by neilkakkar — APPROVED** [review]
Assuming cloud changes will fix the cloud tests, LGTM!

**Comment by EDsCODE** [comment]
Will restore cloud workflow after this is merged

**Pull request merged**